api/src/tests/integration/features/component/test_component_feature.py

```diff
@@ -1,14 +1,15 @@
 from starlette.status import HTTP_200_OK
 
 from common.components import COMPONENT_IDS
+from entities.ComponentResponse import ComponentName
 
 
 def test_components_feature(test_app):
     response = test_app.get("/api/v1/components")
     results = response.json()
-    # json turns ints to str, need to convert back for testing:
-    # using sets as sets dont care about ordering
-    assert {key for key in results.keys()} == set(COMPONENT_IDS.keys())
-    for key, value in results.items():
-        assert value == COMPONENT_IDS[key]
+    components = results["components"]
+    # using sets as sets do not care about ordering
+    assert {key for key in components.keys()} == set(COMPONENT_IDS.keys())
```

Comment:
```suggestion
    assert set(components.keys()) == set(COMPONENT_IDS.keys())
```
This will work right?

Reply:
Yes.